packages/react-scripts/template/README.md

```diff
@@ -1009,6 +1010,12 @@ Learn more about React Storybook:
 
 You can turn your React app into a [Progressive Web App](https://developers.google.com/web/progressive-web-apps/) by following the steps in [this repository](https://github.com/jeffposnick/create-react-pwa).
 
+## Git Repository
```

Comment:
To be honest I think it's fine if don't specifically document this. I appreciate that you took the time to write it down but I don't think it's strictly necessary.